C_CPP/OOPs in C++/Constructor & Destructor/Advanced-Destructor.md

```diff
@@ -0,0 +1,107 @@
+# Destructor Concepts
+
+#### What is a destructor ?
+  Destructors are basically used for the **deallocation of memory when object goes out of scope** and it is used for class object and it's class members. Destructor has the same name as that of class name but it's preceded with ~(tilde).     There can be only one destructor in a class.    
+  There should be no return type and no parameters.
+
+ ### Syntax for using Destructor:
+ Suppose if class name is Add(),
```

Comment:
update it with " Suppose if class name is Add(),
 then ~add()  will deallocate the memory "

Reply:
I have made the changes that you have asked me to change and saved it sir.
Thank you sir.